cfgov/v1/atomic_elements/molecules.py

```diff
@@ -18,7 +18,7 @@ class HalfWidthLinkBlob(blocks.StructBlock):
 
     class Meta:
         icon = 'link'
-        template = '_includes/molecules/half-width-link-blob.html'
+        template = '_includes/molecules/link-blob.html'
```

Comment:
I'm going to leave it as it exist for now because it might cause some confusion. We still have the concept of the `half-width` link blob in Wagtail.